fe/fe-core/src/main/java/com/starrocks/staros/StarMgrServer.java

```diff
@@ -111,6 +112,10 @@ public StarOSBDBJEJournalSystem getJournalSystem() {
         return journalSystem;
     }
 
+    public HttpService getHttpService() {
+        return starMgrServer.getHttpService();
```

Comment:
be aware that it could be possible the StarMgrServer module is disabled, so this should be an optional and can be easily disabled. And when the StarMgrServer is disabled, the http action should not be registered at all.

Reply:
Added cluster mode checking logic to decide if this action need to register